source/agora/test/Base.d

```diff
@@ -202,7 +203,7 @@ public NetworkT makeTestNetwork (NetworkT : TestNetwork)
             c => c.key_pair.address.toString()).array);
         foreach (idx, ref conf; configs)
         {
-            net.apis ~= net.createNewNode(node_configs[idx].key_pair.address.toString(), conf);
+            net.apis[node_configs[idx].key_pair.address] = net.createNewNode(node_configs[idx].key_pair.address.toString(), conf);
```

Comment:
```d
const address = node_configs[idx].key_pair.address;
net.apis[address] = net.createNewNode(address.toString(), conf);
```

Reply:
ah it's a shame you didn't do it as a suggestion

Reply:
What's this fancy new tech you're talking about ?